plexapi/myplex.py

```diff
@@ -707,7 +707,7 @@ def delete(self):
         self._server.query(key, self._server._session.delete)
 
 
-def _connect(cls, url, token, timeout, results, i):
+def _connect(cls, url, token, timeout, fast, results, i, connected_event=None):
```

Comment:
If X_PLEX_ENABLE_FAST_CONNECT is a global configuration option. Should we instead always use fast=True or false based on that, rather than passing it in as an argument?